docs/bucket/notifications/README.md

```diff
@@ -145,7 +148,7 @@ Install an MQTT Broker from [here](https://mosquitto.org/).
 
 ### Step 1: Add MQTT endpoint to Minio
 
-the default location of Minio server configuration file is ``~./minio/config.json``. The MQTT configuration is location in the `mqtt` key under the `notify` top-level key. Create a configuration key-value pair here for your MQTT instance. The key is a name for your MQTT endpoint, and the value is a collection of key-value parameters described in the table below.
+The Minio server configuration file is stored on the backend in json format. The MQTT configuration is location in the `mqtt` key under the `notify` top-level key. Create a configuration key-value pair here for your MQTT instance. The key is a name for your MQTT endpoint, and the value is a collection of key-value parameters described in the table below.
```

Comment:
typo. `The MQTT configuration is location` should be `The MQTT configuration is located`

Reply:
fixed